packages/dashboard-frontend/src/components/EditorSelector/Gallery/index.tsx

```diff
@@ -20,6 +20,7 @@ export type Props = {
   defaultEditorId: string;
   editors: che.Plugin[];
   selectedEditorId: string | undefined;
+  selectorConfig: { showDeprecated: boolean; hideById: string[] } | undefined;
```

Comment:
Would you mind changing the name of this field?

```suggestion
  editorsVisibilityConfig: { showDeprecated: boolean; hideById: string[] } | undefined;
```

Reply:
I have fixed it.